mm/mm_heap/mm.h

```diff
@@ -256,6 +256,14 @@ struct mm_heap_s
 typedef CODE void (*mm_node_handler_t)(FAR struct mm_allocnode_s *node,
                                        FAR void *arg);
 
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+/* All zero allocated pointers to a global unique ptr */
+
+extern uintptr_t g_zero_alloc;
```

Comment:
Global variables should not be used!!!  This will not work properly in kernel mode.  The value should be allocated from the heap.

Reply:
Done. Removed global variable.